app/uk/gov/hmrc/cgtpropertydisposals/util/StringOps.scala

```diff
@@ -21,6 +21,7 @@ object StringOps {
   implicit class StringOps(private val s: String) extends AnyVal {
 
     def removeAllWhitespaces(): String = s.replaceAll(" ", "")
+    def removeNewLineChars(): String   = s.replaceAll("\r", "").replaceAll("\n", "")
```

Comment:
Am wondering if it would better to just escape the characters than replace with blank string. that way there is no alteration to the content and the confusion above of whether we need to change the deserialisation aswell.